test/specs/events.spec.js

```diff
@@ -1,56 +1,224 @@
-describe('Events listeners state', function() {
-
-  const chartConfig = {
-    type: 'scatter',
-    options: {
-      animation: false,
-      scales: {
-        x: {
-          display: false,
-          min: 0,
-          max: 10
-        },
-        y: {
-          display: false,
-          min: 0,
-          max: 10
+describe('Common events', function() {
+
+  describe('state', function() {
+    const chartConfig = {
+      type: 'scatter',
+      options: {
+        animation: false,
+        plugins: {
+          legend: false,
+          annotation: {
+            annotations: [
+              {
+                type: 'line',
+                scaleID: 'y',
+                value: 3
+              },
+              {
+                type: 'line',
+                scaleID: 'y',
+                value: 3,
+                enter: () => true
+              },
+              {
+                type: 'line',
+                scaleID: 'y',
+                value: 3
+              }
+            ]
+          }
         }
-      },
-      plugins: {
-        legend: false,
-        annotation: {
-          annotations: [
-            {
-              type: 'line',
-              scaleID: 'y',
-              value: 3
-            },
-            {
-              type: 'line',
-              scaleID: 'y',
-              value: 3,
-              enter: () => true
-            },
-            {
-              type: 'line',
-              scaleID: 'y',
-              value: 3
-            }
-          ]
+      }
+    };
+    it('should detect listened and moveListened state set to true', function() {
```

Comment:
This was labelled "should not throw any exception" before, but it was not testing that.
Does removing this test reduce coverage? Because we should not write tests for internal state.
Tests for internal state end up being maintenance burden, not improving quality at all.

Reply:
OKI!